keep/api/routes/metrics.py

```diff
@@ -11,6 +16,7 @@
 
 @router.get("")
 def get_metrics(
+    labels: List[str] = Query(None),
```

Comment:
labels are being passed in query params?

Reply:
Yep! This config puts them in query params: https://github.com/keephq/keep/pull/1981/files#diff-39fb6ed0278128b5bc3a74f116aea666a8cf428aa761130b9793dfe17d15ac64R48

Reply:
I don't know if it's a good practice + it's limited by the number of items you can pass in query